card/mod/account/set/right/password.rb

```diff
@@ -38,8 +38,8 @@ def ok_to_read
 end
 
 format :html do
-  view :core do
-    "<em>#{render_raw}</em"
+  view :core, wrap: :em do
```

Comment:
cool.

Reply:
If I were to override a view like this, could I get rid of the wrapping?

Reply:
Aren't these view tags set independent?

Reply:
Looked closer. You extended the `cache` mechanism to handle these, which imo was the right way to go.  That means they're associated with the set and implemented as methods and thus overridable. So I guess all we need is to support `wrap: :none` or something if we want to be able to get rid of wrapping.  (can mark as resolved)